webui/react/src/components/DatePickerFilter.tsx

```diff
@@ -0,0 +1,30 @@
+import { DatePicker } from 'antd';
+import { Moment } from 'moment';
+import React from 'react';
+
+import css from './DatePickerFilter.module.scss';
+import Label from './Label';
+
+interface Props {
+  label: string;
+  onChange?: (date: Moment | null) => void;
+  value?: Moment;
```

Comment:
**chore:** we want to preserve the datetime as a string `YYYY-MM-DD[T]hh:mm:ss+zzzz` as long as possible until the very end when we actually render or if the component requires a datetime object. We don't support it now but when the user profile has a preferred timezone specified, we will start taking this info and render the datetime to the user's preferred timezone at the presentation layer/component.

Reply:
Not sure if I should do something about this right now?

Reply:
Discussed with @apizzini , will leave this as is. Thinking is we go forward with DayJS objects normalized to UTC timezone going forward.